infra/cost/avx.py

```diff
@@ -0,0 +1,137 @@
+from pathlib import Path
+from subprocess import Popen, PIPE
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import json
+import os
+import re
+
+from .fpcore import FPCore
+from .runner import Runner
+from .util import double_to_c_str, chunks
+
+unary_ops = ['neg', 'floor', 'sqrt', 'round', 'ceil', 'fabs']
+binary_ops = ['+', '-', '*', '/', 'fmax', 'fmin']
+ternary_ops = ['fma', 'fmsub', 'fnmadd', 'fnmsub']
+
+compiler = 'cc'
+c_flags = ['-g', '-O3', '-mavx', '-mfma']
+# TODO: What does this do?
+ld_flags = ['-lm']
+driver_name = 'main.c'
+time_unit = 'ms'
+
+time_pat = re.compile(f'([-+]?([0-9]+(\.[0-9]+)?|\.[0-9]+)(e[-+]?[0-9]+)?) {time_unit}')
+
+class AVXRunner(Runner):
+    def __init__(
+        self,
+        working_dir: str,
+        herbie_path: str,
+        num_inputs: Optional[int] = 10000,
+        num_runs: int = 100,
+        threads: int = 1
+    ):
+        super().__init__(
+            name='avx',
+            lang='avx',
+            working_dir=working_dir,
+            herbie_path=herbie_path,
+            # TODO: Does that work?
+            # Need four times as many inputs since each element is a 4 element vector
+            num_inputs=num_inputs*4,
```

Comment:
I'm curious why you didn't go the other way. Like round the number of inputs to the nearest multiple of 4 and then group the inputs.